app/models/book.rb

```diff
@@ -4,4 +4,11 @@ class Book < ApplicationRecord
   has_many :book_authorships, dependent: :destroy
   has_many :authors, through: :book_authorships
   scope :newest, ->{order(created_at: :desc)}
+  scope :search_by_tilte, ->(search){search_books search}
```

Comment:
chu cha mạ ơi, ít kì công 
```suggestion
  scope :search_by_tilte, ->(key_title){where("book_title LIKE ?", "%#{key_title}%") if key_title.present?}
```

Reply:
Dạ do robocop nó bắt em lỗi nhiều ký tự trong một dòng đó chị :sweat_smile: :sweat_smile: